### ISSUE TITLE
Enhancements: Introduce 'Show User Name' Button, Address UI Inconsistencies, and Display List of Group Members

### ISSUE THREAD

**Issue by RunTerror** [CCExtractor/beacon#223]
<img src="https://github.com/CCExtractor/beacon/assets/113206442/678c3eda-f66a-40ed-a08d-f395d38f8eac" alt="Screenshot_1707630462" width="400" /> <img src="https://github.com/CCExtractor/beacon/assets/113206442/46c8650c-07e5-49de-903a-74ea4049c907" alt="Screenshot_1707630850" width="400" />

1. Enhance the Create Hike screen by including a list of beacon members.
2. Address UI inconsistencies for improved visual coherence.
3. Implement a switch in the Create Hike form to toggle the functionality of displaying either the user's real name or remaining 
    anonymous.

**Comment by RunTerror** [comment]
@Akshatji800  i want to work on these issues and features.

**Comment by Akshatji800** [comment]
Hey @RunTerror , I have assigned it to you. Make sure you maintain the being anonymous features as well and somehow your new enhancements doesn't override those.

**Comment by RunTerror** [comment]
> Hey @RunTerror , I have assigned it to you. Make sure you maintain the being anonymous features as well and somehow your new enhancements doesn't override those.

Certainly, @Akshatji800  Once the version upgrade PR is merged, I'll start working on this task. Backend changes are ready locally and will be pushed for review, with needed adjustments.
